packages/terra-progress-bar/src/terra-dev-site/doc/progress-bar/ProgressBar.1.doc.jsx

```diff
@@ -36,13 +42,21 @@ const DocPage = () => (
       },
       {
         title: 'Color',
+        description: <Markdown id="progress-bar-color-docs" src={ProgressBarColorDocs} />,
         example: <ProgressBarColor />,
         source: ProgressBarColorSrc,
       },
       {
-        title: 'Gradient',
-        example: <ProgressBarGradient />,
-        source: ProgressBarGradientSrc,
+        title: 'Foreground and Background Color',
+        description: <Markdown id="progress-bar-two-colors-docs" src={ProgressBarTwoColorsDocs} />,
```

Comment:
Nitpick: the docs look redundant with the title + description markdown heading. One of them can be removed

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2211/commits/722971d7cd3633bc01ccb8550b78e5cda75c22ef
I removed the headers in the doc files to maintain a standard title size amongst the examples.